mirgecom/initializers.py

```diff
@@ -339,8 +340,11 @@ class Lump:
     """
 
     def __init__(
```

Comment:
This might be a good use-case for keyword-only args.

Reply:
I am curious about this, but unsure what advantage it offers here.   Can we discuss it further?

Reply:
https://github.com/illinois-ceesd/mirgecom/pull/95/commits/939cb3ff0f71ef3730b9fdeeaceb3bb8f2618b82

Reply:
I like this linking in the commit for addressing each comment.  Is this what you had in mind?

Reply:
Yep, exactly. It helps me a lot.